phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -378,6 +386,20 @@ public void testSecondaryIndex() throws Exception {
         }
     }
 
+    public static void checkVerifiedStatus(Connection conn, String indexFullName, byte[] rowKey) throws SQLException, IOException {
```

Comment:
ImmutableIndexIT has a similar method (verifyRowsForEmptyColValue). Could we reuse it here?

Reply:
I've merged all three similar methods that I could find.